### PR TITLE
Edits on section 1 and 2

### LINKED ISSUE #1348: Rewrite Introduction and Core Classes sections of Prezi3
Splitting from #1258, which motivates [ha!] the issue but is a specific normative change.

This issue is to rewrite the first two sections of the Presentation API to take into account the following current deficiencies:

- [ ] Be neutral regarding the types of content presented, rather than being heavily image-focused (as was appropriate for previous versions).
- [ ] Make Annotations a core resource type

(Others?)

### PR DESCRIPTION
**Pull request by azaroth42**
Closes #1348

### CODE REVIEW

File: source/api/presentation/3.0/index.md [hunk 1]
```diff
@@ -47,38 +47,38 @@ This is a work in progress and may change without any notices. Implementers shou
...
+Note that in the following descriptions, "object" is used to refer to the object that has been digitized or a born-digital compound object, and "resources" refer to the digital resources and content that are the result of that digitization or digital creation process.
 
 ###  1.2. Motivating Use Cases
 
-There are many different types of digitized or digital compound objects, from ancient scrolls to modern newspapers, from medieval manuscripts to online comics, and from large maps to small photographs. Many of them bear texts, sometimes difficult to read either due to the decay of the physical object or lack of understanding of the script or language.  These use cases are described in a separate [document][prezi-use-case-doc].
+There are many different types of digitized or digital compound objects, from ancient scrolls to modern newspapers, from operas to pop music, and from the earliest black and white film to to computer generated animations. Many of them bear text, sometimes difficult to read either due to the decay of the physical object or lack of understanding of the script or language.  These use cases are described in a separate [document][prezi-use-case-doc].
```
Comment (tomcrane) [inline source/api/presentation/3.0/index.md]: (I think we can work at these examples more, to cover the outer product of [Galleries, Libraries, Archives, Museums] and [Images, Audio, Video] with a few ingenious choices)

A suggestion (not necessarily a requested change):

There are many different types of digitized or digital compound objects; ancient scrolls, paintings, letters, books, newspapers, films, operas, albums, field recordings and computer generated animations. Many of them bear the written or spoken word, sometimes difficult to read or hear either due to the decay of the physical object or lack of understanding of the script or language.

File: source/api/presentation/3.0/index.md [hunk 2]
```diff
@@ -99,17 +99,17 @@ This specification makes use of the following primary resource types:
...
 
 ##### Canvas
 {: #overview-canvas}
 
-A virtual container that represents a page or view and has content resources associated with it or with parts of it. The Canvas provides a frame of reference for the layout of the content. The concept of a Canvas is borrowed from standards like PDF and HTML, or applications like Photoshop and Powerpoint, where the display starts from a blank display and images, video, text and other resources are "painted" on to it.
+A virtual container that represents a particular view of the object and has content resources associated with it or with parts of it. The Canvas provides a frame of reference for the layout of the content, both spatially and temporally. The concept of a Canvas is borrowed from standards like PDF and HTML, or applications like Photoshop and Powerpoint, where the display starts from a blank display and images, video, text and other resources are "painted" on to it.
```
Comment (tomcrane) [inline source/api/presentation/3.0/index.md]: ..., where an initially blank display surface has images, video, text and other content "painted" onto it.

File: source/api/presentation/3.0/index.md [hunk 3]
```diff
@@ -123,26 +123,25 @@ Content resources such as images, audio, video or text that are associated with
...
 {: #overview-annotationpage}
 
-An ordered list of Annotations in a single response, typically associated with a single Canvas, and can be part of an AnnotationCollection.
+An ordered list of Annotations in a single response, typically associated with a single Canvas, and can be part of an Annotation Collection.
 
 ##### Annotation
 {: #overview-annotation}
 
-Content resources and commentary are associated with a Canvas via an Annotation.  This provides a single, coherent method for aligning information, and provides a standards based framework for distinguishing parts of resources and parts of Canvases.  As Annotations can be added later, it promotes a distributed system in which publishers can align their content with the descriptions created by others.
+Content resources, commentary and other references are associated with a Canvas via Annotations.  This provides a single, coherent method for aligning information, and provides a standards based framework for distinguishing parts of resources and parts of Canvases.  As Annotations can be added later, it promotes a distributed system in which publishers can align their content with the descriptions created by others.
```
Comment (tomcrane) [inline source/api/presentation/3.0/index.md]: Start this sentence with "All"?

Comment (tomcrane) [reply source/api/presentation/3.0/index.md]: Or something like... 

All content is linked to the Canvas via Annotations. The same mechanism is used for the visible or audible resources as is used for transcriptions, commentary, tags and other content. This provides a single, coherent...

I know this isn't adding anything new. Just reinforcing the point.

File: source/api/presentation/3.0/index.md (hunk 1 above)
Comment (azaroth42) [reply source/api/presentation/3.0/index.md]: :+1:

File: source/api/presentation/3.0/index.md (hunk 2 above)
Comment (azaroth42) [reply source/api/presentation/3.0/index.md]: :+1:

File: source/api/presentation/3.0/index.md (hunk 3 above)
Comment (azaroth42) [reply source/api/presentation/3.0/index.md]: :+1:

**Comment by azaroth42** [comment]
Made the changes suggested by Tom.

**Comment by jpstroop** [comment]
Will leave for @tomcrane

File: source/api/presentation/3.0/index.md [hunk 4]
```diff
@@ -99,17 +99,17 @@ This specification makes use of the following primary resource types:
...
 
 ##### Canvas
 {: #overview-canvas}
 
-A virtual container that represents a page or view and has content resources associated with it or with parts of it. The Canvas provides a frame of reference for the layout of the content. The concept of a Canvas is borrowed from standards like PDF and HTML, or applications like Photoshop and Powerpoint, where the display starts from a blank display and images, video, text and other resources are "painted" on to it.
+A virtual container that represents a particular view of the object and has content resources associated with it or with parts of it. The Canvas provides a frame of reference for the layout of the content, both spatially and temporally. The concept of a Canvas is borrowed from standards like PDF and HTML, or applications like Photoshop and Powerpoint, where an initally blank display surface has images, video, text and other content "painted" on to it.
```
Comment (tomcrane) [inline source/api/presentation/3.0/index.md]: Powerpoint -> PowerPoint; initally -> initially

File: source/api/presentation/3.0/index.md [hunk 5]
```diff
@@ -47,38 +47,38 @@ This is a work in progress and may change without any notices. Implementers shou
 
 ##  1. Introduction
 
-Access to image-based resources is fundamental to many research disciplines, scholarship and the transmission of cultural knowledge. Digital images are a container for much of the information content in the Web-based delivery of digitized cultural heritage objects. Collections of born-digital images can also benefit from a standardized method to structure their layout and presentation, such as slideshows, image carousels, web comics, and more.
+Access to digital representations of structured resources is a fundamental requirement for many research pursuits, the transmission of cultural knowledge and for the daily pursuits of every web citizen. Digital content is the primary mode of transmission for access to cultural heritage, science and entertainment. Collections of both digitized physical objects and born-digital content can benefit from a standardized description of their structure, layout and presentation mode.
```
Comment (tomcrane) [inline source/api/presentation/3.0/index.md]: repetition of _pursuits_: how about "many research activities"?
"can benefit" -> just "benefit" (stronger)

File: source/api/presentation/3.0/index.md [hunk 6]
```diff
@@ -47,38 +47,38 @@ This is a work in progress and may change without any notices. Implementers shou
...
 
-This document describes how the structure and layout of a complex image-based object can be made available in a standard manner. Many different styles of viewer can be implemented that consume the information to enable a rich and dynamic experience, consuming content from across collections and hosting institutions.
+This document describes how the structure and layout of a composite objects can be made available in a standard manner. Many different styles of viewer can be implemented that consume the information to enable a rich and dynamic experience, consuming content from across collections and institutions.
```
Comment (tomcrane) [inline source/api/presentation/3.0/index.md]: "layout of a composite objects" -> "...object" or "layout of composite objects"
second use of "consuming" ->  "presenting content from across..."?

File: source/api/presentation/3.0/index.md [hunk 7]
```diff
@@ -47,38 +47,38 @@ This is a work in progress and may change without any notices. Implementers shou
...
 
 ### 1.1. Objectives and Scope
 
-The objective of the IIIF (pronounced "Triple-Eye-Eff") Presentation API is to provide the information necessary to allow a rich, online viewing environment for primarily image-based objects to be presented to a human user, likely in conjunction with the [IIIF Image API][image-api]. This is the sole purpose of the API and therefore the descriptive information is given in a way that is intended for humans to read, but not semantically available to machines. In particular, it explicitly does __not__ aim to provide metadata that would drive discovery of the digitized objects.
+The objective of the IIIF (pronounced "Triple-Eye-Eff") Presentation API is to provide the information necessary to allow a rich, online viewing environment for structured digital objects to be presented to a human user, likely in conjunction with the [IIIF Image API][image-api]. This is the sole purpose of the API and therefore the descriptive information is given in a way that is intended for humans to read, but not semantically available to machines. In particular, it explicitly does __not__ aim to provide metadata that would drive discovery of the objects.
```
Comment (tomcrane) [inline source/api/presentation/3.0/index.md]: "likely in conjunction" -> "often in conjunction"?
Not for now, but the phrase about not driving discovery was I think misunderstood - will try to find evidence of this.

**Pull request merged**